src/tests/ftest/util/server_utils.py

```diff
@@ -245,49 +292,49 @@ def stop_server(setname=None, hosts=None):
         print("<SERVER> Exception occurred: {0}".format(str(error)))
         raise ServerFailed("Server didn't stop!")
 
     if not hosts:
         return
 
-    # make sure they actually stopped
-    # but give them some time to stop first
+    # Make sure the servers actually stopped.  Give them time to stop first
+    # pgrep exit status:
+    #   0 - One or more processes matched the criteria.
+    #   1 - No processes matched.
+    #   2 - Syntax error in the command line.
+    #   3 - Fatal error: out of memory etc.
     time.sleep(5)
-    found_hosts = []
-    for host in hosts:
-        proc = subprocess.Popen(["ssh", host,
-                                 "pgrep '(daos_server|daos_io_server)'"],
-                                stdout=subprocess.PIPE)
-        stdout = proc.communicate()[0]
-        resp = proc.wait()
-        if resp == 0:
-            # a daos process was found hanging around!
-            found_hosts.append(host)
-
-    if found_hosts:
-        kill_server(found_hosts)
-        raise ServerFailed("daos processes {} found on hosts "
-                           "{} after stop_server() were "
-                           "killed".format(', '.join(stdout.splitlines()),
-                                           found_hosts))
+    result = cluster_cmd(
+        hosts, "pgrep '(daos_server|daos_io_server)'", False, expect_rc=1)
+    if len(result) > 1 or 1 not in result:
+        bad_hosts = [
+            node for node in list(result[key]) for key in result if key != 1]
+        kill_server(bad_hosts)
+        raise ServerFailed(
+            "DAOS server processes detected after attempted stop on {}".format(
+                ", ".join([str(result[key]) for key in result if key != 1])))
 
     # we can also have orphaned ssh processes that started an orted on a
     # remote node but never get cleaned up when that remote node spontaneiously
     # reboots
     subprocess.call(["pkill", "^ssh$"])
 
+
 def kill_server(hosts):
-    """
-    Sometimes stop doesn't get everything.  Really whack everything
-    with this.
+    """Forcably kill any daos server processes running on the specified hosts.
+
+    Sometimes stop doesn't get everything.  Really whack everything with this.
 
-    hosts -- list of host names where servers are running
+    Args:
+        hosts (list): list of host names where servers are running
     """
-    kill_cmds = ["pkill '(daos_server|daos_io_server)' --signal INT",
-                 "sleep 5",
-                 "pkill '(daos_server|daos_io_server)' --signal KILL"]
-    for host in hosts:
-        subprocess.call("ssh {0} \"{1}\"".format(host, '; '.join(kill_cmds)),
-                        shell=True)
+    kill_cmds = [
+        "pkill '(daos_server|daos_io_server)' --signal INT",
+        "sleep 5",
+        "pkill '(daos_server|daos_io_server)' --signal KILL",
+    ]
+    # Intentionally ignoring the exit status of the command
+    cluster_cmd(hosts, "; ".join(kill_cmds), False)
```

Comment:
To avoid this message from being printed:
`boro-[41,43,52-53]: failure running 'pkill '(daos_server|daos_io_server)' --signal INT; sleep 5; pkill '(daos_server|daos_io_server)' --signal KILL': rc=1
`
```suggestion
    cluster_cmd(hosts, "; ".join(kill_cmds), False, None, None)
```